data/tokens.json

```diff
@@ -389,7 +325,7 @@
       "caption": {
         "type": "typography",
         "value": {
-          "fontFamily": "{typography.family.base}",
+          "fontFamily": "{typography.fontFamily.mulish}",
```

Comment:
Do we want to explicitly call this Mulish? I'm thinking maybe we abstract it out to `fontFamily.sansSerif` or a more generic name, so we could theoretically change things in the repo without having to re-do instances within the codebase if we delete this variable

Reply:
works for me, updated